src/documents/tutorial-userInterfaceOptions/UserInterfaceOptions.md

```diff
@@ -4,62 +4,100 @@ layout: default
 * instantiates a cookie-based [Settings Store](../SettingsStore.md) for storing the user's preferences; and
 * acts upon the user's preferences.
 
 ![Screen shot of the UI Options Component](../images/uio-showcase.png "Screen shot of the UI Options Component")
 
-This tutorial assumes that:
-
-* you are already familiar with HTML, Javascript and CSS,
-* you are familiar with what the UI Options preferences editor is and does, and
-* now you just want to know how to add it to your website.
+You can try the [UI Options demo here](http://build.fluidproject.org/infusion/demos/uiOptions/).
 
 ## Scenario
 
-You're putting together a website that you know will have a diverse audience. You'd like to allow your visitors to
-customize the presentation of the site to their individual needs, by enlarging the text or increasing the visual
-contrast, for example. This tutorial will show you how to add the Infusion [UI
-Options](http://build.fluidproject.org/infusion/demos/uiOptions/) component to your site.
+You are putting together a website and would like to give visitors a way to customize the website presentation to their individual needs. This tutorial will show you how to add the Infusion UI Options component to your site.
 
 These are the basic steps to add UI Options to your website:
 
-1. [Download and install the Infusion library](#download-and-install-the-ui-options-library)
+1. [Download and install UI Options](#download-and-install-the-ui-options-library)
 2. [Prepare your page](#prepare-your-page)
     * [The Sliding Panel](#the-sliding-panel)
     * [The Table of Contents](#the-table-of-contents)
 3. [Add dependencies to the page](#add-dependencies-to-the-page)
 4. [Add the UI Options component](#add-the-ui-options-component)
 
-The rest of this tutorial will explain each of these steps in detail.
+The rest of this tutorial will explain these steps and assumes that you can work with HTML, JavaScript and CSS.
 
 ## Download and install the UI Options library
 
-1. [Download the UI Options library](https://github.com/fluid-project/infusion/releases/download/v2.0.0/infusion-uiOptions-2.0.0-source.zip)
-  (there is also a [minified version UI Options](https://github.com/fluid-project/infusion/releases/download/v2.0.0/infusion-uiOptions-2.0.0-minified.zip)
-   for deployment purposes).
-2. Unzip the contents of the downloaded ZIP file to a location within your project. This will result in a new directory
-   `infusion`.<div class="infusion-docs-note"><strong>Note:</strong> In this guide we will use the directory
-   `my-project/lib/`.</div>
-3. Your `infusion` folder will include a single file containing all of the JavaScript you need
+Get the latest Infusion release:
+* [Zip package of an Infusion release](https://github.com/fluid-project/infusion/releases)
+* Using npm: `npm i infusion`
+
+If you prefer trying the latest development source code, you can visit [Infusion on GitHub](https://github.com/fluid-project/infusion).
+
```

Comment:
I'm not sure this summary is needed as you go into detail about all of these below. If you want the summary maybe make them links to the related sections instead.